### PR TITLE
Update sl to map to the java action inputs

### PR DESCRIPTION
**Pull request by Tirla-Alin**
This should be merged after the pull request on the cs-actions is merged.
https://github.com/CloudSlang/cs-actions/pull/183

### CODE REVIEW

File: content/io/cloudslang/base/ssh/ssh_command.sl [hunk 1]
```diff
@@ -61,7 +79,40 @@ operation:
...
+          required: false
+          private: true
+      - allowed_ciphers:
+          default: 'aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc'
+          required: false
+      - allowedCiphers:
+          default: ${get("allowed_ciphers", "")}
+          required: false
+          private: true
       - timeout: '90000'
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_command.sl]: @Tirla-Alin Please set here default: '9000' and required: false

File: content/io/cloudslang/base/ssh/ssh_command.sl [hunk 2]
```diff
@@ -61,7 +79,40 @@ operation:
...
           required: false
           private: true
+      - private_key_data:
+          required: false
+      - privateKeyData:
+          default: ${get("private_key_data", "")}
+          required: false
+          private: true
+      - known_hosts_policy:
+          default: allow
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_command.sl]: @Tirla-Alin Please use '' for default value allow

File: content/io/cloudslang/base/ssh/ssh_command.sl (hunk 1 above)
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_command.sl]: @Tirla-Alin Please set the default and optional tags for port and pty as well.

**Comment by Bonczidai** [comment]
please also put a link in the description - `the pull request` -> `CloudSlang/cs-actions#180` or direct link

File: content/io/cloudslang/base/ssh/ssh_command.sl [hunk 3]
```diff
@@ -78,8 +135,35 @@ operation:
...
+          sensitive: true
+          required: false
+      - proxyPassword:
+          default: ${get("proxy_password", "")}
+          sensitive: true
+          required: false
+          private: true
     java_action:
-      gav: 'io.cloudslang.content:cs-ssh:0.0.34'
+      gav: 'io.cloudslang.content:cs-ssh:0.0.36'
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_command.sl]: @Tirla-Alin Please set gav version to 0.0.35

**Comment by tethryus** [comment]
@Tirla-Alin Please update gav to io.cloudslang.content:cs-ssh:0.0.35  in validate_linux_machine_ssh_access with the new inputs, the new descriptions, the gav release and the test files (including the new inputs and a use case for proxy). Before merging please run the changes in the builder with the -des tag.

File: content/io/cloudslang/base/ssh/ssh_flow.sl [hunk 4]
```diff
@@ -66,29 +84,38 @@ flow:
...
+      - known_hosts_policy:
+          default: 'allow'
+          required: false
+      - known_hosts_path:
+          required: false
+      - allowed_ciphers:
+          default: 'aes128-ctr,aes128-cbc,3des-ctr,3des-cbc,blowfish-cbc,aes192-ctr,aes192-cbc,aes256-ctr,aes256-cbc'
           required: false
-          private: true
       - timeout: '90000'
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_flow.sl]: @Tirla-Alin Please set the default and required tags for the timeout input to reflect optional and default value in the description.

File: content/io/cloudslang/base/ssh/ssh_flow.sl [hunk 5]
```diff
@@ -66,29 +84,38 @@ flow:
...
-          default: ${get("close_session", "false")}
-          private: true
       - agent_forwarding:
           required: false
-      - agentForwarding:
-          default: ${get("agent_forwarding", "")}
-          required: false
-          private: true
       - smart_recovery: "True"
       - retries: '5'
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_flow.sl]: @Tirla-Alin Please update retries input with the default value to reflect the description.

File: content/io/cloudslang/base/ssh/ssh_flow.sl [hunk 6]
```diff
@@ -66,29 +84,38 @@ flow:
...
-      - closeSession:
-          default: ${get("close_session", "false")}
-          private: true
       - agent_forwarding:
           required: false
-      - agentForwarding:
-          default: ${get("agent_forwarding", "")}
-          required: false
-          private: true
       - smart_recovery: "True"
```
Comment (tethryus) [inline content/io/cloudslang/base/ssh/ssh_flow.sl]: @Tirla-Alin Please update the smart_recovery input with the default value, to reflect description.

File: test/io/cloudslang/base/ssh/ssh_flow.inputs.yaml [hunk 7]
```diff
@@ -74,4 +74,31 @@ testSSHWorksWhenPTYtrue:
...
+  result: SUCCESS
+
+testSSHWorksWithProxy:
+  inputs:
+    - host: localhost
+    - port: "49153"
+    - command: "echo testing\nexit\n"
+    - username: root
+    - password: screencast
+    - proxy_host: <proxy_host>
```
Comment (tethryus) [inline test/io/cloudslang/base/ssh/ssh_flow.inputs.yaml]: @Tirla-Alin Does this test work when the proxy_host is used in order to connect to the host?

Comment (Tirla-Alin) [inline test/io/cloudslang/base/ssh/ssh_flow.inputs.yaml]: Yes.

**Comment by tethryus** [comment]
@Tirla-Alin 👍 Great work. Can be merged

**Pull request merged**